test/test_importer.py

```diff
@@ -768,6 +774,10 @@ def test_set_fields(self):
             album.load()
             self.assertEqual(album.genre, genre)
             self.assertEqual(album.collection, collection)
+            for item in album.items():
+                self.assertEqual(item.genre, genre)
+                self.assertEqual(item.collection, collection)
+                self.assertEqual(item.comments, comments)
```

Comment:
Have you checked that this test fails without the above changes to `set_fields`? I suspect that due to https://github.com/beetbox/beets/pull/2988, this part of the test was already successful before; it doesn't verify that the item tags are acutally set, though.

Maybe change to

```suggestion
            for item in album.items():
                self.assertEqual(item.get("genre", with_album=False), genre)
                self.assertEqual(item.get("collection", with_album=False), collection)
                self.assertEqual(item.get("comments", with_album=False), comments)
```
```
```

Reply:
Yes, I checked. It failed without the change. In particular, it would not set the comments. Nevertheless I have adopted your suggestion for a stricter `get`.